bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/handler/DaikinMadokaHandler.java

```diff
@@ -121,7 +129,30 @@ public void initialize() {
             submitCommand(new GetPowerstateCommand()); // always keep the "GetPowerState" aftern the "GetOperationMode"
             submitCommand(new GetSetpointCommand());
             submitCommand(new GetFanspeedCommand());
-        }, 10, c.refreshInterval, TimeUnit.SECONDS);
+            submitCommand(new GetCleanFilterIndicatorCommand());
+
+            // As it is a complex operation - it has been extracted to a method.
+            retrieveOperationHours();
+
+            submitCommand(new GetEyeBrightnessCommand());
+        }, new Random().nextInt(30), c.refreshInterval, TimeUnit.SECONDS); // We introduce a random start time, it
+                                                                           // avoids when having multiple devices to
+                                                                           // have the commands sent simultaneously.
+    }
+
+    private void retrieveOperationHours() {
+        // This one is special - and MUST be ran twice, after being in priv mode
+        // run it once an hour is sufficient... TODO
+        submitCommand(new EnterPrivilegedModeCommand());
+        submitCommand(new GetOperationHoursCommand());
+        // a 1second+ delay is necessary
+        try {
+            Thread.sleep(1500);
+        } catch (InterruptedException e) {
+            logger.error("Error in retrieveOperationHours()", e);
+            Thread.currentThread().interrupt();
+        }
```

Comment:
If the thread was interrupted you should try and return as soon as possible since an interrupt means that the system wants your thread to shutdown.

Simply returning early from this may not be sufficient as the method's caller might continue operating as if things are normal. I suggest having the InterruptedException continue up the stack and let the caller handler it.